logproof/src/test.rs

```diff
@@ -176,3 +180,241 @@ pub fn bfv_delta<const N: usize>(coeff_modulus: ZqRistretto, plaintext_modulus:
     let limbs = delta.as_limbs().map(|l| l.into());
     Uint::<N>::from_words(limbs[0..N].try_into().unwrap())
 }
+
+/**
+ * Generate a lattice problem for the BFV scheme.
+ */
+pub fn seal_bfv_encryption_linear_relation<B, const N: usize>(
+    message: u64,
+    degree: u64,
+    plain_modulus: u64,
+    batch_encoder: bool,
```

Comment:
This isn't quite the shape I would expect, but not sure it matters much since we'll replace it anyways.

Reply:
It is just a test function I pulled out to use in both the linked proof and the SDLP tests. It's just a test function, not a general API.